src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -1202,15 +1202,50 @@ func getTempUrls(securityScopedUrls: [URL]) -> [URL] {
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Upload to a specified bucket
+
+You can also perform an upload operation to a specific bucket by providing the `bucket` option. This option can either be a string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.
+
+See [additional storage bucket usage](/[platform]/build-a-backend/storage/set-up-storage/#additional-storage-bucket-usage) for more information.
+
+```ts title="Provide bucket as a string"
+import { uploadData } from 'aws-amplify/storage';
+
+try {
+  const result = uploadData({
+    path: "album/2024/1.jpg",
+    data: file,
+    options: {
+      bucket: "assignedNameInAmplifyBackend"
+    }
+  }).result;
+} catch (error) {
+  console.log(`Error: ${error}`)
+}
+```
+```ts title="Provide bucket as an object"
+options: {
```

Comment:
lets provide full code example. The other way has backfired on us before. but we can keep it absolute required plus the option. Same everywhere

Reply:
I've made this change. I agree that the full examples have less potential to confuse readers.